### DISCUSSION TITLE
Keep track of all of the missed/slow words for an all-time practice

### DISCUSSION THREAD

**Discussion by oriongonza** [monkeytypegame/monkeytype#4855] [Ideas]
As of right now you can only practice words for the latest session. I'd like to have a practice mode for all time missed words.

I can also give this a shot if I can get some guidance

**Comment by Miodec** [comment]
Its planned for the future

**Reply by oriongonza** [reply]
Is there something that I could work on to speed this up? I really want this feature

**Reply by Miodec** [reply]
Ive got some specific thoughts in mind and some more things need to be changed before working on this feature, sorry.

**Comment by isaacweber613** [comment]
I would also love to have this feature so i don't have to only practice missed words at the end of the session i would be able to practice all time missed words. PLEASE implement this feature ASAP, thanks.

**Reply by oriongonza** [reply]
Hey - this is free software, people are working on this in their free time.
If you really need this feature go ahead an implement it yourself, but don't demand features from people that are doing this on their spare time,
